model-engine/tests/unit/api/test_llms.py

```diff
@@ -156,15 +156,13 @@ def test_completion_sync_endpoint_not_found_returns_404(
     assert response_1.status_code == 404
 
 
-# When enabling this test, other tests fail with "RunTumeError got Future <Future pending> attached to a different loop"
-# https://github.com/encode/starlette/issues/1315#issuecomment-980784457
-@pytest.mark.skip(reason="Need to figure out FastAPI test client asyncio funkiness")
-def test_completion_stream_success(
+@pytest.mark.asyncio
```

Comment:
Is it safe to enable this test? What happened to the previous issue?

Reply:
after modifying the body of the test to leverage `async` ContextManagers (ensuring that any asynchronous generators properly call `aclose` prior to garbage collection, which I suspect may have been the issue), and leveraging the new `get_async_test_client_wrapper` fixture I defined in `conftest.py`, I was able to run `just unit-test` on this entire file with no issues.

Reply:
Ah, nvm - just noticed its affecting tests defined in other files, I misunderstood the initial comment.

Reply:
I fixed the weird interaction between these tests by defining an event_loop fixture in `tests/unit/api/conftest.py` that ensures each async test runs with its own event loop, and reconfiguring `test_create_streaming_task_success` as an async test. cc @yunfeng-scale, I think you were the last to take a look at this issue?

Reply:
lgtm!